api/controllers/app/install/controller.go

```diff
@@ -164,11 +171,21 @@ func NewInstallController(opts ...InstallControllerOption) (*InstallController,
 	}
 
 	if controller.appReleaseManager == nil {
-		appReleaseManager, err := appreleasemanager.NewAppReleaseManager(
-			*controller.releaseData.AppConfig,
+		var appReleaseManagerOpts []appreleasemanager.AppReleaseManagerOption
+		appReleaseManagerOpts = append(appReleaseManagerOpts,
 			appreleasemanager.WithLogger(controller.logger),
 			appreleasemanager.WithReleaseData(controller.releaseData),
 		)
+
+		// Add registry settings if available
+		if controller.registrySettings != nil {
```

Comment:
appReleaseManagerOpts and this nil check is probably unnecessary as WithRegistrySettings accepts a pointer